Directory.Build.props

```diff
@@ -4,19 +4,16 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
         <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
 
     <PropertyGroup>
-        <NoWarn>$(NoWarn),SA0001,CA1014</NoWarn>
+        <NoWarn>$(NoWarn),SA0001,SA1200,SA1516,CA1014,CA1812</NoWarn>
         <SimpleExecVersion>6.1.0</SimpleExecVersion>
         <NuGetVersion>4.1.0</NuGetVersion>
         <OctokitVersion>0.31.0</OctokitVersion>
     </PropertyGroup>
 
-    <ItemGroup>
-        <PackageReference Remove="Microsoft.CodeAnalysis.FxCopAnalyzers" />
-    </ItemGroup>
```

Comment:
This was redundant. We no longer had this package in the parent Directory.Build.props